spec/rails_helper.rb

```diff
@@ -3,6 +3,7 @@
 require 'spec_helper'
 require File.expand_path('../../config/environment', __FILE__)
 require 'rspec/rails'
+require 'shoulda/matchers'
```

Comment:
It doesn't work without it.
Shoulda needs knowing about which test framework we're using.
That's we require the gem following rspec-rails in rails_helper.rb.
Probably, the other way of configuring is https://github.com/thoughtbot/shoulda-matchers#configuration